index.js

```diff
@@ -0,0 +1,14 @@
+
+// conservative limit for now
+const TENANT_ID_REGEX = new RegExp(/^[a-zA-Z][a-zA-Z-_]{0,98}[a-zA-Z]$/);
+const UUIDV4_REGEX = new RegExp(/^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i); 
+const ISO_DATETIME_REGEX = new RegExp(/^(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))$/);
+const HEX_CODE_REGEX = new RegExp(/^#(([0-9a-fA-F]{2}){3}|([0-9a-fA-F]){3})$/);
```

Comment:
Can we rename this to indicate it is for validating hex colour codes, as the name is a bit confusing at the moment.

I do also wonder which scenarios we want to validate a hex colour code as if we are accepting css/html style colours we should probably accept the other colour formats too.

```suggestion
const HEX_COLOUR_CODE_REGEX = new RegExp(/^#(([0-9a-fA-F]{2}){3}|([0-9a-fA-F]){3})$/);
```

Reply:
ill update the name sec